src/client/pages/VergunningDetail/EigenParkeerplaats.tsx

```diff
@@ -11,70 +15,71 @@ export function EigenParkeerplaats({
 }) {
   const isVerleend = vergunning.processed && vergunning.decision === 'Verleend';
   const isAfgehandeld = vergunning.processed;
-  let location1 = null;
-  let location2 = null;
-
-  if (Array.isArray(vergunning.locations)) {
-    [location1, location2] = vergunning.locations;
-  }
 
   return (
     <>
       <InfoDetail label="Kenmerk" value={vergunning.identifier || '-'} />
-      <InfoDetail label="Verzoek" value={vergunning.requestType} />
-
-      {!!location1 && (
-        <>
-          <Location
-            label="Adres"
-            location={`${location1.street} ${location1.houseNumber}`}
-          />
-          <InfoDetail label="Soortplek" value={location1.type} />
-          <InfoDetail
-            label="Parkeervak"
-            value={
-              <Linkd href={location1.url} external={true}>
-                Bekijk parkeervak
-              </Linkd>
-            }
-          />
-        </>
-      )}
+      <InfoDetail
+        label="Verzoek"
+        valueWrapperElement="div"
+        value={
+          <ul>
+            {vergunning.requestTypes.map((d, i) => (
+              <li key={d}>{d}</li>
+            ))}
+          </ul>
+        }
+      />
 
-      {!!location2 && (
-        <>
-          <Location
-            label="Adres"
-            location={`${location2.street} ${location2.houseNumber}`}
-          />
-          <InfoDetail label="Soortplek" value={location2.type} />
-          <InfoDetail label="Parkeervak" value={location2.url} />
-        </>
-      )}
+      <InfoDetailGroup>
+        {vergunning.locations?.map((location, i) => {
+          return (
+            <div
+              className={i > 0 ? styles2.InfoGroupDivider : ''}
+              key={JSON.stringify(location)}
+            >
+              <Location
+                label={`Adres ${vergunning.locations?.length == 2 ? i + 1 : ''}`}
```

Comment:
je kunt hier beter de `i` gebruiken. Je wilt toch alléén bij de eerste locatie "Adres" neerzetten, bij elke volgende een nummer erbij. De check op `locations.length` in de loop suggereert dat dit speciale logica is. Bij 3 locaties zou je dan: Adres, Adres, Adres krijgen, iemand die niet weet dat er altijd maar 2 locaties zijn zou dan denken: huh?!

`i > 0 ? i + 1 : ''` Maakt altijd: Adres, Adres 2, Adres 3

Reply:
Ja maar vond het expliciet maken wel mooier bij 2 adressen, zodat je dan ook begint met `Adres 1` erbij wel mooier :) Dus nu zijn de opties:
Ik had idd length >1 kunnen doen, zodat het daarna ook Adres 3 zou pakken. Maar aangezien we er max 2 krijgen leek me dit ook goed en extra expliciet.

Reply:
Als je dat doet hoef je die check er niet bij te zetten en kun je gewoon `Adres ${i+1}` doen.